.github/spellcheck/.spellcheck.yml

```diff
@@ -7,12 +7,24 @@ matrix:
     - .github/spellcheck/.wordlist.txt
     encoding: utf-8
   pipeline:
-  - pyspelling.filters.markdown:
+  - pyspelling.filters.markdown:  
   - pyspelling.filters.html:
       comments: false
       ignores:
       - code
       - pre
+      - script
+      - style
+      - a
+      - img
+      - href
+      - meta
+      - svg
+      - title      
+      - text
+  - pyspelling.filters.text:
   sources:
   - '**/*.md'
+  exclude:
+  - '**/GOVERNANCE.md'
```

Comment:
(A) This does not seem to be having its intended effect: https://github.com/7h3-3mp7y-m4n/kubestellar/actions/runs/11997351426/job/33442851201#step:4:414 still shows a complaint.

(B) Is the real problem here the fact that the file is a broken symlink?